### PR TITLE
Add new analyser_merge for emergency access point dataset in Luxembourg

### PR DESCRIPTION
**Pull request by DavidMoraisFerreira**
Hello everyone,
This PR contains the implementation for an analyser_merger that imports [the dataset](https://data.public.lu/fr/datasets/linstallation-des-points-de-secours-rettungspunkte/) of all emergency access points in Luxembourg.

Issues/Questions:

1. In `missing_official` and `possible_merge` I was unsure what `id` should be set to. Furthermore, I think that this analyser_merge needs a new `item` number. Do I need `update_official`? I am a bit confused about what these different methods imply.
2. Should I be using the dynamic merger? 

@grischard Do you have any suggestions and/or change requests?

Looking forward to getting this merged into Osmose :)

### CODE REVIEW

**Comment by frodrigo** [comment]
For `update_official` you need a ref. But you does not setup one: `osmRef =`

dynamic merger: no, it is for merge multiple types of objects from one data source.

File: analysers/analyser_merge_emergency_points_LU.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+
+class Analyser_Merge_Emergency_Points_LU(Analyser_Merge):
+    def __init__(self, config, logger=None):
+        Analyser_Merge.__init__(self, config, logger)
+        self.missing_official = self.def_class(item=9110, id=3, level=3, tags=['merge'],
+                                               title=T_('Emergency point not integrated'))
+        self.possible_merge = self.def_class(item=9111, id=3, level=3, tags=['merge'],
```
Comment (frodrigo) [inline analysers/analyser_merge_emergency_points_LU.py:31]:
```suggestion
        self.missing_official = self.def_class(item=8440, id=3, level=3, tags=['merge'],
                                               title=T_('Emergency point not integrated'))
        self.possible_merge = self.def_class(item=8441, id=3, level=3, tags=['merge'],
```

**Comment by DavidMoraisFerreira** [comment]
Is there a documentation that describes missing_official and other methods in more detail? I found the source code documentation a bit minimalistic.
The goal here is to have an analyser_merger from the official dataset to osm.

**Comment by frodrigo** [comment]
This part of the doc is not written, yet.

**Comment by DavidMoraisFerreira** [comment]
It seems like merging existing objects in OSM with existing objects in the upstream dataset isn't working. Please don't merge the PR yet :)

File: analysers/analyser_merge_emergency_points_LU.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+
+class Analyser_Merge_Emergency_Points_LU(Analyser_Merge):
+    def __init__(self, config, logger=None):
+        Analyser_Merge.__init__(self, config, logger)
+        self.missing_official = self.def_class(item=8440, id=3, level=3, tags=['merge'],
+                                               title=T_('Emergency point not integrated'))
+        self.possible_merge = self.def_class(item=8441, id=3, level=3, tags=['merge'],
+                                               title=T_('Emergency point integration suggestion'))
```
Comment (frodrigo) [inline analysers/analyser_merge_emergency_points_LU.py:32]:
```suggestion
        self.missing_official = self.def_class(item=8440, id=1, level=3, tags=['merge'],
                                               title=T_('Emergency point not integrated'))
        self.possible_merge = self.def_class(item=8441, id=3, level=3, tags=['merge'],
                                               title=T_('Emergency point integration suggestion'))
```

**Comment by DavidMoraisFerreira** [comment]
Thank you, it seems to be working now. The nodes already integrated aren't being created anymore.

**Comment by frodrigo** [comment]
Thank you

**Comment by frodrigo** [comment]
First results available
http://osmose.openstreetmap.fr/fr/errors/?item=8440,8441&useDevItem=all

If it is ok for you, web make it public.

**Comment by DavidMoraisFerreira** [comment]
Thanks for the feedback:
I found an example which seems odd:
http://osmose.openstreetmap.fr/en/map/#item=xxxx&useDevItem=all&zoom=18&lat=49.649148&lon=6.224924&level=3&tags=&fixable= and http://osmose.openstreetmap.fr/en/map/#item=xxxx&useDevItem=all&zoom=18&lat=49.649148&lon=6.224924&level=3&tags=&fixable=

There is one integration suggestion and one new integration, is this expected behaviour? If so, then I am OK with the results :)

**Comment by frodrigo** [comment]
> There is one integration suggestion and one new integration, is this expected behaviour? If so, then I am OK with the results :)

Yes

**Comment by frodrigo** [comment]
@jocelynj we can enable items 8440 and 8441

**Comment by DavidMoraisFerreira** [comment]
Was there an update to enable 8440 and 8441? If I can help by creating a PR on osmose-frontend, let me know. In the meantime, I imported the dataset manually, so the interest of having this AnalyserMerge is to keep OSM<>Dataset up-to-date.

**Comment by frodrigo** [comment]
It is here and running, but just hidden by default
But for now nothing to show : http://osmose.openstreetmap.fr/fr/errors/?item=8440,8441&useDevItem=all
All to 0.

cc @jocelynj

**Pull request merged**